app/controllers/request-to-go-live/organisation-address/get.controller.js

```diff
@@ -8,12 +8,15 @@ const { response } = require('../../../utils/response')
 const { countries } = require('@govuk-pay/pay-js-commons').utils
 const formatServicePathsFor = require('../../../utils/format-service-paths-for')
 const { getAlreadySubmittedErrorPageData } = require('../../stripe-setup/stripe-setup.util')
+const { isSwitchingCredentialsRoute } = require('../../../utils/credentials')
 
 module.exports = function getOrganisationAddress (req, res) {
   const isRequestToGoLive = Object.values(paths.service.requestToGoLive).includes(req.route && req.route.path)
+  const isStripeUpdateOrgDetails = req.url ? req.url.startsWith('/your-psp/') : false
```

Comment:
Can we use Boolean() here?

```suggestion
  const isStripeUpdateOrgDetails = Boolean(req.url ? req.url.startsWith('/your-psp/'))
```

Reply:
Good idea - thanks